src/components/navbar/navLinkSection/NavLinkSection.module.scss

```diff
@@ -0,0 +1,34 @@
+@import '../../../utils/variables';
+
+.link {
+    margin: 10px;
+    text-decoration: none;
+    font-weight: $semi-bold;
+    color: $blue;
+    text-align: center;
+    font-size: calc(14px + (24 - 14) * ((100vw - 320px) / (1920 - 320)));
+    transition: color 0.4s;
+    cursor: pointer;
+
+    @media (min-width: 2222px) {
+        font-size: 27px;
+    }
+    @media (min-width: $max-width) {
+        margin: 10px 20px;
+    }
+    @media (max-width: 985px) {
+        font-size: 15px;
+    }
+    @media (max-width: 886px) {
+        margin: 10px 5px;
+    }
+    @media (max-width: 824px) {
```

Comment:
These `max-widths` seem pretty random. Can you use the current screen size variables only please.

Reply:
they are the sizes where they break and need to be adjusted.

Reply:
Yeah, I don't know what best for this kind of thing? Is it better to have 3/4 sizes that your website is designed for or to scale things as they break? Probably worth some thought. 

After 30 seconds of thinking, I feel like for the designers its better if we just have like 3 sizes that the website conforms to and they can design for all those. 
But then from the dev side, I don't think its ever that simple so being flexible is probably better??

TLDR I don't really know what's best for this kind of thing... We should discuss but don't worry for now